api-get-object-attributes.go

```diff
@@ -0,0 +1,155 @@
+package minio
+
+import (
+	"context"
+	"encoding/xml"
+	"errors"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/minio/minio-go/v7/pkg/encrypt"
+	"github.com/minio/minio-go/v7/pkg/s3utils"
+)
+
+// ObjectAttributesOptions ...
+// https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObjectAttributes.html
+type ObjectAttributesOptions struct {
+	ObjectAttributes     string
+	VersionID            string
+	MaxParts             int
+	PartNumberMarker     int
```

Comment:
I would kind of expect us to handle these internally and not expose these in the API.

So we just return all parts... I think this is overengineering on AWS' part already. We shouldn't make users bother with these and just return all parts.